src/lib/core/CHIPConfig.h

```diff
@@ -1210,6 +1210,19 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MINMDNS_DYNAMIC_OPERATIONAL_RESPONDER_LIST 0
 #endif // CHIP_CONFIG_MINMDNS_DYNAMIC_OPERATIONAL_RESPONDER_LIST
 
+/*
+ * @def CHIP_CONFIG_MINMDNS_MAX_PARALLEL_RESOLVES
+ *
+ * @brief Determines the maximum number of parallel SRV processing.
```

Comment:
```suggestion
 * @brief Determines the maximum number of SRV records that can be processed in parallel.
```

Assuming that's accurate?